jwst/srctype/srctype.py

```diff
@@ -53,11 +53,11 @@ def set_source_type(input_model):
             user_type = input_model.meta.target.source_type_apt
             log.info(f'Input SRCTYAPT = {user_type}')
             if user_type is None:
-                log.warning(f'SRCTYAPT keyword not found in input; using SRCTYPE instead')
+                log.warning('SRCTYAPT keyword not found in input; using SRCTYPE instead')
```

Comment:
So just for my own knowledge, you don't include the 'f' formatting specifier when it's just a plain text string with no values from variables to be inserted?

Reply:
harsh pep8 checking *oof*

Reply:
Correct.  It doesn't result in an error, but it adds extra processing time because the interpreter has to treat it as an f-string, and look for expressions to evalutate at runtime.  `flake8` considers that on par with unnecessary imports and other "cruft" in code.

```
In [1]: %timeit foo = "This is a nice string"
12.8 ns ± 0.14 ns per loop (mean ± std. dev. of 7 runs, 100000000 loops each)

In [2]: %timeit foo = f"This is a nice string"
13.1 ns ± 0.134 ns per loop (mean ± std. dev. of 7 runs, 100000000 loops each)

In [3]: %timeit foo = f"This is a nice {'string'}"
71.6 ns ± 0.36 ns per loop (mean ± std. dev. of 7 runs, 10000000 loops each)
```